src/otx/v2/adapters/datumaro/adapter/datumaro_dataset_adapter.py

```diff
@@ -410,7 +413,7 @@ def _filter_unlabeled_data(self, unlabeled_dataset: DatumDataset, unlabeled_file
         for item in copy_dataset:
             if item.id not in file_list:
                 unlabeled_dataset.remove(item.id, item.subset)
-
+    
```

Comment:
typo?

Reply:
Ah, this comes from my VIM plugin :) , done -> https://github.com/openvinotoolkit/training_extensions/pull/2631/commits/1cbcdcd953906dadb225f5d365962c2dc9ba8467